apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,50 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, signal, WritableSignal } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities()"
+      customClass="bg-light-blue"
+      (AddNewRecordEmitter)="handleAddNewCity()"
+      [ListItemTemplate]="itemTemplate">
+      <img src="assets/img/city.png" width="200px" image />
+      <ng-template #itemTemplate let-item>
+        <app-list-item [item]="item">
+          <p>{{ item.name }}</p>
+        </app-list-item>
+      </ng-template>
+    </app-card>
+  `,
   standalone: true,
-  imports: [],
+  imports: [CardComponent, ListItemComponent],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities: WritableSignal<City[]> = signal([]);
+  constructor(
+    private http: FakeHttpService,
+    private store: CityStore,
+  ) {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((s) => this.store.addAll(s));
+
+    this.store.cities$.subscribe((s) => this.cities.set(s));
```

Comment:
no prefer declarative over imperative.

you can transform your state to be signal or use toSignal l.31

Reply:
done